modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp

```diff
@@ -149,144 +166,32 @@ else {
 		onlyActions="<%= true %>"
 	/>
 
-	<li class="control-menu-nav-item">
-		<div class="d-none d-sm-block dropdown">
-			<button aria-haspopup="true" class="component-action dropdown-toggle taglib-icon" data-toggle="liferay-dropdown">
-				<aui:icon cssClass="<%= StringPool.BLANK %>" image="ellipsis-v" markupView="lexicon" />
-
-				<span class="sr-only">
-					<liferay-ui:message key="options" />
-				</span>
-			</button>
-
-			<ul class="dropdown-menu dropdown-menu-right" role="menu">
-				<li>
-					<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutSetRevisions" onclick="<portlet:namespace />openSitePagesVariationsDialog();">
-						<liferay-ui:message key="site-pages-variation" />
-					</a>
-				</li>
+	<portlet:renderURL var="layoutRevisionStatusURL" windowState="<%= LiferayWindowState.EXCLUSIVE.toString() %>">
+		<portlet:param name="mvcPath" value="/view_layout_revision_status.jsp" />
+	</portlet:renderURL>
 
-				<c:if test="<%= !layoutRevision.isIncomplete() && !layout.isTypeContent() %>">
-					<li>
-						<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutRevisions" onclick="<portlet:namespace />openPageVariationsDialog();">
-							<liferay-ui:message key="page-variations" />
-						</a>
-					</li>
-					<li>
-						<a class="dropdown-item" href="javascript:Liferay.fire('<portlet:namespace />viewHistory', {layoutRevisionId: '<%= layoutRevision.getLayoutRevisionId() %>', layoutSetBranchId: '<%= layoutRevision.getLayoutSetBranchId() %>'}); void(0);" id="viewHistoryLink">
```

Comment:
Hi @BarbaraCabrera , could you please keep this IDs, as they are reference points for Functional tests / Poshi tests?